src/services/formatting/smartIndenter.ts

```diff
@@ -343,28 +343,28 @@ namespace ts.formatting {
 
         export function getContainingList(node: Node, sourceFile: SourceFile): NodeArray<Node> | undefined {
             if (node.parent) {
-                return getListByRange(node.getStart(sourceFile), node.getEnd(), node.parent);
+                return getListByRange(node.getStart(sourceFile), node.getEnd(), node.parent, sourceFile);
             }
             return undefined;
         }
 
-        function getListByPosition(pos: number, node: Node): NodeArray<Node> | undefined {
+        function getListByPosition(pos: number, node: Node, sourceFile: SourceFile): NodeArray<Node> | undefined {
             if (!node) {
                 return;
             }
-            return getListByRange(pos, pos, node);
+            return getListByRange(pos, pos, node, sourceFile);
         }
 
-        function getListByRange(start: number, end: number, node: Node): NodeArray<Node> | undefined {
+        function getListByRange(start: number, end: number, node: Node, sourceFile: SourceFile): NodeArray<Node> | undefined {
             switch (node.kind) {
                 case SyntaxKind.TypeReference:
-                    return getListIfVisualStartEndIsInListRange((<TypeReferenceNode>node).typeArguments, start, end, node);
+                    return getListIfVisualStartEndIsInListRange((<TypeReferenceNode>node).typeArguments, start, end, node, sourceFile);

```

Comment:
You could create a closure so you don't have to repeat the arguments `start, end, node, sourceFile` everywhere.